sympy/geometry/ellipse.py

```diff
@@ -1436,7 +1437,11 @@ def __new__(cls, *args, **kwargs):
             elif len(args) == 2:
                 # Assume (center, radius) pair
                 c = Point(args[0], dim=2)
-                r = sympify(args[1])
+                r = Tuple(args[1], )
+                r = r.xreplace(dict(
+                [(f, simplify(nsimplify(f, rational=True)))
+                 for f in r.atoms(Float)]))
```

Comment:
I think this may support `evaluate` as an option, as in `Point`
https://github.com/sympy/sympy/blob/68aa33a78433c4c5e8308e9d2dc096e1524f9e6b/sympy/geometry/point.py#L165-L170

Reply:
I think it will become more complicated and we have to add an extra argument evaluate.

Reply:
I think it needs some way to stop `float`s being converted to `Rational`. I think it would be better if `evaluate` defaulted to `False` for `Point` as well.

Reply:
But I think it will be better to change to Rational otherwise due to floating value it may rounded and result will always not going to be good and  in above case we can't round the floating value.So I don't think there should be an option for not conveting floating value to Rational.

Reply:
If we follow that argument to its conclusion then all floats *everywhere* should be converted to Rational. That doesn't solve the potential problems that users have though: if they use floats they can get rounding errors before calling any of Sympy's functions. Users who want exact results should provide exact inputs. Since `Point` already automatically converts float to Rational (by default) then maybe it makes sense to do so here as well. I wonder if actually the proper fix is that neither should be converted to Rational and `is_tangent` should be changed to use floating point comparison with tolerance.

Reply:
See [https://github.com/sympy/sympy/issues/11484](https://github.com/sympy/sympy/issues/11484). There may be other issues but I found that from a quick search.

Reply:
>  `is_tangent` should be changed to use floating point comparison with tolerance.

I think this can be a possibility. But you know floating error even can occur in exact no. provided by the user like if the user provides 2.6666666 then there is a possibility that it may rounded in the calculation. And for floating point comparison with tolerance, we can do but the question is up to which degree we should do?

Reply:
See also #6716